Dockerfile

```diff
@@ -1,5 +1,5 @@
-FROM node:16-buster-slim
-RUN apt update && apt upgrade -y && apt install ffmpeg git -y
+FROM node:16-alpine
+RUN apk update && apk upgrade && apk add ffmpeg git && rm -rf /var/cache/apk/*
```

Comment:
What's that "`rm -rf /var/cache/apk/`" for?

Reply:
Is used to clean the internal cache of alpine, in order to keep the image smaller.